index.html

```diff
@@ -1,9 +1,9 @@
 ---
 layout: default
 
-link: Tell them to accept Bitcoin Cash
+link: Tweet to them
 link_mobile: TWEET THEM
-workonit_tweet: Please consider accepting Bitcoin Cash as a payment option @TWITTERHANDLE- it’s fast and secure, with low fees!
+workonit_tweet: "Please consider accepting #bitcoinCash for payments @TWITTERHANDLE - it’s fast and secure, with low fees! More @ "
```

Comment:
I would use #BitcoinCash

Reply:
Agreed, though I did write it originally with the lower case B, I am just used to camel casing it....

When we use this text it adds out url to it?

Reply:
It does append our URL to the end, yep.

Reply:
Perfect then. Can we change it to #BitcoinCash and then I'd accept it.